### ISSUE TITLE
Link does not appear in Link array after edit message

### ISSUE THREAD

**Issue by the-itz** [arminfro/participator#9]
It seems that a link appears in the link list only after the creation of a message.
If a message has been edited and a link been added during this, it does not appear in the link array.
@arminfro is this in your garden?

**Comment by arminfro** [comment]
True, this hasn't been implemented yet. It's my garden, yes :wink:
